README.md

```diff
@@ -2,7 +2,9 @@
 
 Answers to common technical questions from .NET open source project owners and maintainers.
 
-**[The Book of .NET OSS website](https://jamesnk.github.io/BookOfDotNetOSS/)**
+~**[The Book of .NET OSS website](https://jamesnk.github.io/BookOfDotNetOSS/)**~
+
+**Important:** This repository is being replaced by a PR to docs.microsoft.com - [Open-source Library Guidance PR](https://github.com/Microsoft/dotnet-blog/pull/305)
```

Comment:
This link seems to be broken. It looks like the `Microsoft/dotnet` repository doesn't exist. I had a quick search but couldn't see where it was supposed to point. 😕

Reply:
Huh, how did that link get there? Will fix, thanks.